### DISCUSSION TITLE
Is it possible to link a pdf and a markdown file so that annotations are always pasted in that markdown file?

### DISCUSSION THREAD

**Discussion by tprotopopescu** [RyotaUshio/obsidian-pdf-plus#214] [Q&A] [answered]
Is it possible to link a pdf and a markdown file so that annotations are always pasted in that markdown file? I had the impression that maybe that is what the PDF property might be able to do, but as far as I can see that just allows you to use other properties in your markdown file in your annotation links (as per https://github.com/RyotaUshio/obsidian-pdf-plus/discussions/144). 

I use autopaste and I find that quite often my annotations end up in different files (I often work on more than one file, so the 'last pasted' or 'last active' tends to change). However, for each pdf I would like annotations to get pasted only to the one file I designate. Is this possible at the moment? Could the PDF property be used to do this?

**Comment by JacobH140** [comment]
hi, were you able to figure out a workflow here?

**Comment by RyotaUshio** [comment, marked as answer]
Sorry for the late reply. If things go well, this will be supported in v1.0.0 (see https://github.com/RyotaUshio/obsidian-pdf-plus/issues/258).
